INTEGRATE.md

```diff
@@ -0,0 +1,31 @@
+# Local integration between Porta/System with Keycloak through Zync
+Install these 3 components to have a complete integration of managed SSO applications in Porta.
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+To have a keycloak running locally, do `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+
+Finally [configure Red Hat Single Sign-On](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
```

Comment:
I think this is redundant with the last section

Reply:
@hallelujah It is not the same, this is `2.3.1. Configure Red Hat Single Sign-On` and the other one is `2.3.2. Configure 3scale` and it needs to be done in this order 🤔